config/application.yml.default

```diff
@@ -99,6 +99,9 @@ doc_auth_supported_country_codes: '["US", "GU", "VI", "AS", "MP", "PR", "USA" ,"
 doc_auth_vendor_randomize: false
 doc_auth_vendor_randomize_alternate_vendor: ''
 doc_auth_vendor_randomize_percent: 0
+doc_auth_vendor_default: 'lexis_nexis'
```

Comment:
Do we use/need this?

Reply:
Right now? Technically no, but that true of the whole PR 😉

Reply:
@solipet
can you please share what this variable does? 
currently we have the similarly named `doc_auth_vendor` variable which serves a different function which I think can cause confusion

to me it seems the a/b options here are Socure Capture App vs our ReactApp? what are your thought about this flag naming leaning towards the capture experience?

Reply:
In my mind, `doc_auth_vendor_default` would eventually replace `doc_auth_vendor` as I think it's more descriptive of its intent - in the absence of an enabled A/B test, this is the default so use it. Currently, `doc_auth_vendor` sounds like it names the vendor in use, but that may be overridden by the A/B test.

I chose not to simply rename the var as I wasn't looking to disturb the status quo. Having now looked at where it is used, it's trivial to just rename it so I'm inclined to do that. I would add the default var (and use it) in one PR, then once that is deployed remove the older var. Sound good?

Reply:
hi @solipet - 

in your implementation, it appears  we have 3 vendors and 2 doc_auth A/B testing mechanisms with 2 buckets each:
1. (mock, lexisnexis)
2. (socure, lexisnexis)

now that we have 3 vendors (mock, socure, and lexis_nexis), I think it would be simpler to maintain having 1 doc_auth A/B test  but with 3 buckets [mock, socure, lexis_nexis].  We would then use the percentage values per bucket to control the routing

Reply:
So I've taken over the DOC_AUTH_VENDOR a/b test and removed the randomize vars. Originally, the test was between lexisnexis and acuant. For sandbox envs, the default is `mock` and the test is disabled. For staging/prod, default is `lexis_nexis` and the test let's us set a percentage to go to Socure.